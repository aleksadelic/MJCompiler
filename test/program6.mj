program MJProgram
	int matrix[][];
	int i, j;
{

	void main() {
		
		matrix = new int[2][3];
		
		i = 0;
		while (i < 2) {
			j = 0;
			while (j < 3) {
				matrix[i][j] = i * 3 + j + 1;
				j++;
			}
			i++;
		}
		
		i = 0;
		while (i < 2) {
			j = 0;
			while (j < 3) {
				print(matrix[i][j]);
				j++;
			}
			print(eol);
			i++;
		}
		
		print(eol);
		matrix[0][0]++;

		i = 0;
		while (i < 2) {
			j = 0;
			while (j < 3) {
				print(matrix[i][j]);
				j++;
			}
			print(eol);
			i++;
		}
		
		read(matrix[1][0]);
		read(matrix[1][1]);
		read(matrix[1][2]);
		
		print(eol);
		matrix[0][0]++;

		i = 0;
		while (i < 2) {
			j = 0;
			while (j < 3) {
				print(matrix[i][j]);
				j++;
			}
			print(eol);
			i++;
		}
		
	}
	
}