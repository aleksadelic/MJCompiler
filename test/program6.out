    1    2    3    
    4    5    6    
    
    2    2    3    
    4    5    6    
    
    3    2    3    
    1    2    3    

Completion took 13155 ms